CHANGELOG.md

```diff
@@ -1,5 +1,66 @@
 # Logic Apps Designer
 
+# <<<<<<< HEAD
+
+## [2.65.0](https://github.com/Azure/LogicAppsUX/compare/v2.64.0...v2.65.0) (2023-08-31)
+
+### Bug Fixes
+
+- **designer:** SimpleQueryBuilder Negatory Operators Serialization/Deserializing Properly ([#3232](https://github.com/Azure/LogicAppsUX/issues/3232)) ([3512b15](https://github.com/Azure/LogicAppsUX/commit/3512b1509b2f14b48e067354ccbc6ee18d97c075))
```

Comment:
If customer-facing, changelog comments should be worded with language that the customer can understand - please consider rewording the bug fix descriptions:

```suggestion
- **Designer**: SimpleQueryBuilder Negatory Operators Serialization/Deserializing Properly ([#3232](https://github.com/Azure/LogicAppsUX/issues/3232)) ([3512b15](https://github.com/Azure/LogicAppsUX/commit/3512b1509b2f14b48e067354ccbc6ee18d97c075))
```